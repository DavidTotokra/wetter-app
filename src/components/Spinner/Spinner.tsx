import { Content } from "./style";

export function Spinner() {
  return (
    <Content>
    <div className="lds-spinner">
      <div></div><div></div><div></div>
      <div></div> <div></div><div></div>
      <div></div><div></div><div></div>
    </div>
    </Content>
  );
}