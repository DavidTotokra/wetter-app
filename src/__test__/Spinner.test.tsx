

import { render } from '@testing-library/react'
import { Spinner } from '../components/Spinner/Spinner'

describe('Spinner Component', () => {
  it('Should render correctly', () => {
    const Sinner = render(<Spinner />)
    expect(Sinner.container).toMatchSnapshot()
  })
})