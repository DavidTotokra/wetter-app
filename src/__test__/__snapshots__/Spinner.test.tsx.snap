// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component Should render correctly 1`] = `
<div>
  <div
    class="sc-bczRLJ iByUCn"
  >
    <div
      class="lds-spinner"
    >
      <div />
      <div />
      <div />
      <div />
       
      <div />
      <div />
      <div />
      <div />
      <div />
    </div>
  </div>
</div>
`;
